app/imports/templates/components/nameStatus.js

```diff
@@ -161,19 +161,21 @@ Template['components_nameStatus'].helpers({
       var revealDeadline = Math.floor(new Date().getTime()/1000) + 48 * 60 * 60;      
       return Names.find({registrationDate: {$gt: revealDeadline}, name:{$gt: ''}},{}).count() > 0;
     },
-    averageValue() {
-      var average = _.reduce(
-          Names.find({value: {$gt:0.01}}).fetch(), function(memo,num) { 
-            return memo + num.value; 
-          }, 0);
-      return Math.round(average*100/Names.find({value: {$gt:0.01}}).count())/100 || '--';
+    medianValue() {
+      var disputedNames = Names.find({value: {$gt:0.01}}, {sort: {value: 1}}).fetch();
```

Comment:
I'm adding an unrelated new bit where it limits the amount of registered names it keeps an eye on to 50.